### PR TITLE
Aside implemented

### PR DESCRIPTION
**Pull request by akshayvinchurkar1**

### CODE REVIEW

**Comment by 0crat** [comment]
@filfreire please, pay attention to this pull request

**Comment by 0crat** [comment]
Job `gh:yegor256/tacit#137` is now in scope.

**Comment by 0crat** [comment]
Job `gh:yegor256/tacit#137` assigned to @filfreire  ([profile](http://www.0crat.com/u/filfreire})). The budget is [fixed](http://datum.zerocracy.com/pages/policy.html#4) and it is 30 minutes. Please, read the [Policy](http://datum.zerocracy.com/pages/policy.html) and go ahead.

**Comment by 0crat** [comment]
[@filfreire](http://www.0crat.com/u/filfreire) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
[@filfreire](http://www.0crat.com/u/filfreire) this job was assigned to you 8 days ago. It will be taken away from you soon, unless you close it, see [§8](http://datum.zerocracy.com/pages/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by 0crat** [comment]
@filfreire[/z](http://www.0crat.com/u/filfreire) resigned from [#137](https://github.com/yegor256/tacit/issues/137), please stop working

**Comment by 0crat** [comment]
The job [#137](https://github.com/yegor256/tacit/issues/137) assigned to @filfreire[/z](http://www.0crat.com/u/filfreire). The budget is 30 minutes, see [§4](http://datum.zerocracy.com/pages/policy.html#4). Please, read [§8](http://datum.zerocracy.com/pages/policy.html#8) and [§9](http://datum.zerocracy.com/pages/policy.html#9). If the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html).

File: index.html [hunk 1]
```diff
@@ -63,7 +68,8 @@ <h2>Forms</h2>
               <option value="male">male</option>
             </select>
             Your age:
-            <label><input type="radio" id="under" name="age" value="under"/> I'm under 18</label>
+            <label><input type="radio" id="unls
+              er" name="age" value="under"/> I'm under 18</label>
```
Comment (filfreire) [inline index.html]: @akshayvinchurkar1 you seem to have a typo here and now the `id` isn't correct:
```
 id="unls
             er"
```

can you kindly:
1) fix this id issue;
2) provide a screenshot of the end result of the aside feature work?

File: index.html [hunk 2]
```diff
@@ -157,10 +163,12 @@ <h2>Lists</h2>
           <dd>A red fruit that is hard to peel.</dd>
         </dl>
         <h2>Blockquote</h2>
+
```
Comment (filfreire) [inline index.html]: @akshayvinchurkar1 also, upon closer inspection, we can remove these empty newlines between elements, so the file is uniform.

**Comment by 0crat** [comment]
The user @filfreire[/z](https://www.0crat.com/u/filfreire) resigned from [#137](https://github.com/yegor256/tacit/issues/137), please stop working

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually or invite more people to the project, as explained in [§51](http://www.zerocracy.com/policy.html#51)

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` are banned at this job; I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` are banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
There is an unrecoverable failure on my side. Please, submit it [here](https://github.com/zerocracy/farm/issues):

```
PID: 4@b0528f24-e874-4ae3-b65f-20504b57f1ed, thread: pool-166-thread-1
org.hamcrest.MatcherAssert[20] java.lang.AssertionError: HTTP response status is not equal to 205:
400 Bad Request [https://api.github.com/notifications]
Server: GitHub.com
Date: Wed, 23 May 2018 02:36:19 GMT
Content-Type: application/json; charset=utf-8
Content-Length: 126
Status: 400 Bad Request
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4971
X-RateLimit-Reset: 1527045485
X-GitHub-Media-Type: github.v3
Access-Control-Expose-Headers: ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
X-Runtime-Rack: 0.041230
X-GitHub-Request-Id: 808E:5156:1AEEEED:458E632:5B04D3A3

{"message":"Problems parsing JSON","documentation_url":"https://developer.github.com/v3/activity/notifications/#mark-as-read"}
Expected: HTTP response with status 205
     but: was <400 Bad Request [https://api.github.com/notifications]
Server: GitHub.com
Date: Wed, 23 May 2018 02:36:19 GMT
Content-Type: application/json; charset=utf-8
Content-Length: 126
Status: 400 Bad Request
X-RateLimit-Limit: 5000
X-RateLimit-Remaining: 4971
X-RateLimit-Reset: 1527045485
X-GitHub-Media-Type: github.v3
Access-Control-Expose-Headers: ETag, Link, Retry-After, X-GitHub-OTP, X-RateLimit-Limit, X-RateLimit-Remaining, X-RateLimit-Reset, X-OAuth-Scopes, X-Accepted-OAuth-Scopes, X-Poll-Interval
Access-Control-Allow-Origin: *
Strict-Transport-Security: max-age=31536000; includeSubdomains; preload
X-Frame-Options: deny
X-Content-Type-Options: nosniff
X-XSS-Protection: 1; mode=block
Referrer-Policy: origin-when-cross-origin, strict-origin-when-cross-origin
Content-Security-Policy: default-src 'none'
X-Runtime-Rack: 0.041230
X-GitHub-Request-Id: 808E:5156:1AEEEED:458E632:5B04D3A3

{"message":"Problems parsing JSON","documentation_url":"https://developer.github.com/v3/activity/notifications/#mark-as-read"}>
```

[1.0-SNAPSHOT](https://github.com/zerocracy/farm): Issue: yegor256/tacit#137, Action: `created`

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by filfreire** [comment]
@akshayvinchurkar1 any chance you can follow-up on the code review for this PR? Thank you 👍

Also, if you don't have the chance, would you mind me reopening this PR and do the improvements I suggested on the CR?

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by 0crat** [comment]
@filfreire[/z](https://www.0crat.com/u/filfreire) everybody who has role `DEV` is banned at [#137](https://github.com/yegor256/tacit/issues/137); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Comment by filfreire** [comment]
@akshayvinchurkar1 I'm closing this PR. Feel free to re-open a new one following-up to my code review comments.

**Comment by 0crat** [comment]
Job `gh:yegor256/tacit#137` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#137](https://github.com/yegor256/tacit/issues/137) is now out of scope

**Pull request closed without merging**